### PR TITLE
minor demo tweaks and ubl=>ubi supporting changes

### PR DESCRIPTION
**Pull request by RasonJ**
updates that likely will break the demo until ubl-ubi interface issues are worked through.

### CODE REVIEW

File: reactivesearch/package-lock.json
Comment (epugh) [inline reactivesearch/package-lock.json:1]: Nice to see thsie updated...

File: reactivesearch/package.json [hunk 1]
```diff
@@ -5,24 +5,38 @@
   "keywords": [],
   "main": "src/index.js",
   "engines": {
-    "node": "<17"
+    "node": "<20"
```
Comment (epugh) [inline reactivesearch/package.json:8]: Much nicer to be on newer Node!

File: reactivesearch/src/ts/index.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ const context = new Context(window, document);
 
 
 /**/
-const writer = new UbiWriter('http://127.0.0.1:9200', 'ubl_log', queryResolver, sessionResolver,  debug);
+const writer = new UbiWriter('http://127.0.0.1:9200', 'bad_log_shouldnt_exist', queryResolver, sessionResolver,  debug);
```
Comment (epugh) [inline reactivesearch/src/ts/index.ts:86]: Not sure I undertstand this change?

File: reactivesearch/webpack.config.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const {CleanWebpackPlugin} = require('clean-webpack-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
```
Comment (epugh) [inline reactivesearch/webpack.config.js:3]: are we actually using NodePolyfillPlugin?

**Review by epugh — CHANGES_REQUESTED** [review]

File: reactivesearch/src/ts/index.ts (hunk 2 above)
Comment (RasonJ) [reply reactivesearch/src/ts/index.ts:86]: Nothing in that file should be firing yet.  It's from the 1st js demo.  So, if we see that index appear, something's going on that shouldn't be.

File: reactivesearch/webpack.config.js (hunk 3 above)
Comment (RasonJ) [reply reactivesearch/webpack.config.js:3]: It's one of the "solutions" for using the OS js Client class with newer versions of reactivesearch.

**Pull request merged**